components/usage/pkg/db/dbtest/user.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package dbtest
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gitpod-io/gitpod/usage/pkg/db"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	"gorm.io/gorm"
+)
+
+type User struct {
+	ID           uuid.UUID      `gorm:"primary_key;column:id;type:char;size:36;"`
+	AvatarURL    string         `gorm:"column:avatarUrl;type:char;size:255;"`
+	Name         string         `gorm:"column:name;type:char;size:255;"`
+	FullName     string         `gorm:"column:fullName;type:char;size:255;"`
+	CreationDate db.VarcharTime `gorm:"column:creationDate;type:varchar;size:255;"`
+
+	// user has more field but we don't care here as they are just used in tests.
+}
+
+func (user *User) TableName() string {
+	return "d_b_user"
+}
+
+func NewUser(t *testing.T, user User) User {
```

Comment:
It's bit of an anti-pattern to have a `testing.T` used in a non test package. Normally we'd pull this into a separate package - it also has the benefit of not being included in the resulting binary if it's in a different package (not referenced by "prod" code)

Reply:
it's in a testing package

Reply:
Sorry, I got confused. The User model definition should be moved out of the `dbtest` package into the `db` package